submission/Motivator/script.js

```diff
@@ -0,0 +1,86 @@
+// Global variables
+const quoteText = document.getElementById('quote-text');
+const quoteAuthor = document.getElementById('quote-author');
+const taskList = document.getElementById('task-list');
+const progressFill = document.getElementById('progress-fill');
+const progressText = document.getElementById('progress-text');
+const addTaskButton = document.getElementById('add-task');
+const newTaskInput = document.getElementById('new-task');
+
+// Motivational Quotes (can add more quotes here)
+const quotes = [
```

Comment:
These are too few quotes. How about fetching quotes from a quotes api? I'm sure you will find plenty such quotes if you search on the internet.